bbl/slides/asciidoc/chapters/technical/vue.adoc

```diff
@@ -0,0 +1,15 @@
+== VueJS
+
+=== Intro
+
+Methods, Props, Lifecycle
+
+=== Store with VueX
+
+=== Router
+
+=== Tooling
```

Comment:
I think we should be move tooling in intro part.
And we can talk about Vue CLI / Vue UI / Vue devtools ... 
How to start a project ... 
?